### PR TITLE
Wizard recipe: libusb-v1.0.23

### PR DESCRIPTION
**Pull request by Gnimuc**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: libusb
* Version: v1.0.23

@staticfloat please review and merge.

### CODE REVIEW

File: L/libusb/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+make -j${nproc}
+make install
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc)
+]
```
Comment (Gnimuc) [inline L/libusb/build_tarballs.jl]: Oops! This should be:
```
platforms = supported_platforms()
```

File: L/libusb/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libusb", :libusb)
```
Comment (Gnimuc) [inline L/libusb/build_tarballs.jl]:
```suggestion
    LibraryProduct("libusb-1.0", :libusb)
```

File: L/libusb/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [p for p in supported_platforms() if !isa(p, FreeBSD)]
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct(["libusb","libusb-1.0"], :libusb)
```
Comment (giordano) [inline L/libusb/build_tarballs.jl]:
```julia
julia> using Pkg.BinaryPlatforms

julia> parse_dl_name_version("libusb-1.0.0.dylib", MacOS())
("libusb-1", v"0.0.0")
```
So you need to add `"libusb-1"` as well:
```suggestion
    LibraryProduct(["libusb", "libusb-1", "libusb-1.0"], :libusb)
```

Comment (Gnimuc) [reply L/libusb/build_tarballs.jl]: Aha!

**Pull request merged**